translations/en-us-all.toml

```diff
@@ -1,7 +1,7 @@
 # en-us translation file for op-bot
 # http://github.com/osprogramadores/op-bot
 
-welcome = "Welcome %s. What your interest in coding? Have fun!\nDon't forget to introduce yourself to the group!"
+welcome = "Welcome %s. Please, introduce yourself. The introduction is a requirement in order to stay in this group. Thank you!"
```

Comment:
@magnussolidus pode mudar por favor para: Please, introduce yourself. You will be removed from the group if this requirement is not met.

Reply:
Feito.